### ISSUE TITLE
Exception accessing login

### ISSUE THREAD

**Issue by mmihaylov** [IdentityServer/IdentityServer3#2109]
Hello, 

I get a lot of random errors like this:

`IdentityServer3.Core.Configuration.Hosting.LogProviderExceptionLogger | Unhandled exception
IdentityServer3.Core.Configuration.Hosting.ErrorPageFilterAttribute | Exception accessing: /IdentityServer3/core/login`

Sometimes the URI is `/IdentityServer3/core/login` and sometimes `/identityserver3/core/login`

Do you know what could be the reason for this?

**Comment by brockallen** [comment]
Who or what is making the requests to that URL? Start there. This is not a lot of info for us to go on.

**Comment by mmihaylov** [comment]
Hi Brock, 
I have a MVC application, which is using Microsoft's OpenIdConnect library to trigger the authentication. IdSrv is installed as a sub application of this website. I have implemented hybrid flow. 

I have decresed the logging level and according to the logs of IdSrv I get the following:

`Info | IdentityServer3.Core.ResponseHandling.UserInfoResponseGenerator | Profile service returned to the following claim types: sub name preferred_username given_name family_name email role
Info | IdentityServer3.Core.Endpoints.UserInfoEndpointController | End userinfo request
Info | IdentityServer3.Core.Results.UserInfoResult | Returning userinfo response.
Info | IdentityServer3.Core.Endpoints.AuthenticationController | Login page requested
Info | IdentityServer3.Core.Endpoints.AuthenticationController | No cookie matching signin id found
Error | IdentityServer3.Core.Configuration.Hosting.LogProviderExceptionLogger | Unhandled exception
Error | IdentityServer3.Core.Configuration.Hosting.ErrorPageFilterAttribute | Exception accessing: /identityserver3/core/login`

This error happens quite randomly and I cannot reproduce it. If needed I will try to provide more information (although not sure exactly what kind).

Regards,

**Comment by mmihaylov** [comment]
Do you think the problem might be related to this Katana bug - https://katanaproject.codeplex.com/workitem/197

**Comment by brockallen** [comment]
Possibly.

**Comment by mmihaylov** [comment]
Hello, 

so this issue happens quite randomly. I open the login page, enter my credentials, submit and then I get again the login page. The only solution I have found so far is to recycle the app pool, but this is not a solution. 

Since we are using load balancer I have set the following properties to IdentityServer app. 

RequireSsl = false
IssuerUri = https://www.mydomain.wtst/IdentityServer3/core
PublicOrigin = https://www.mydomain.wtst/

This is the fiddler log I have saved: https://drive.google.com/file/d/0B1NNodXQjif0cGg0ZHY5cm9ZeEk/view
